developing/dev-setup.mdx

```diff
@@ -18,8 +18,25 @@ $ cargo install refinery_cli
 ```
 
 ### MacOS
+First, install [Homebrew](https://brew.sh/). Then, run the following commands:
+
+```bash
+brew install postgresql
+brew install protobuf
+brew install pnpm
```

Comment:
Do we need any libraries like libssl or are they already included in base macos?